tests/TestCase/Shell/Task/TestTaskTest.php

```diff
@@ -280,22 +280,6 @@ public function testFixtureArrayGenerationFromModel()
         $this->assertEquals($expected, $result);
     }
 
-    /**
-     * test that the generation of fixtures works correctly.
-     *
-     * @return void
-     */
-    public function testFixtureArrayGenerationIgnoreSelfAssociation()
-    {
-        TableRegistry::clear();
-        $subject = new CategoryThreadsTable();
-        $result = $this->Task->generateFixtureList($subject);
-        $expected = [
-            'app.category_threads',
```

Comment:
Does this mean that self associations will attempt to include the child/parent fixtures?